docs/aloe.rst

```diff
@@ -30,4 +30,12 @@ extensively documented here.
 
     Run Python tests as well as Gherkin.
 
+.. option:: -a attr
+
+    Run features and scenarios with the given tag.
```

Comment:
I agree with @danni. You can't assume lettuce users migrating to aloe are familiar with nose. If you don't want to describe the option here, you can mention it in the "porting from lettuce" section.